g_docs.go

```diff
@@ -70,6 +70,7 @@ func generateDocs(curpath string) {
 	}
 
 	rootapi.Infos = swagger.Information{}
+	rootapi.Infos.License = &swagger.License{}
```

Comment:
If you set license here, License will always encode into JSON. As it's not empty now.

https://github.com/astaxie/beego/blob/master/swagger/swagger.go#L48